### PR TITLE
Bootstrap integration tests

### PR DESCRIPTION
**Pull request by csomh**
Signed-off-by: Hunor Csomortáni <csomh@redhat.com>

### CODE REVIEW

File: tests/integration/utils.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+    yields that path, then cleans everything up.
+
+    Args:
+        directory: A path to a directory to archive.
+    Yields:
+        Path to the archive created.
+    Raises:
+        None.
+    """
+    tempdir = tempfile.mkdtemp()
```
Comment (MartinBasti) [inline tests/integration/utils.py]: we run py3 only, so you can use here https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory

```
with TemporaryDirectory as tmpdir_path:
     # do magic
    yield archive
```

File: tox.ini [hunk 2]
```diff
@@ -3,7 +3,24 @@ envlist = py36,py37
...
+deps =
+    pytest
+    requests
+passenv = OMPS_INT_TEST_*
+setenv =
+    REQUESTS_CA_BUNDLE = /etc/pki/tls/certs/ca-bundle.crt
+commands =
+    pytest -vv --confcutdir=tests/integration tests/integration
+
+[flake8]
```
Comment (MartinBasti) [inline tox.ini]: This should be added to `.travis.yml` to run flake in CI

Comment (csomh) [reply tox.ini]: Actually, I'll remove this change for now, and I'll do a proper flake8 setup in a future PR.

File: tests/integration/utils.py (hunk 1 above)
Comment (csomh) [reply tests/integration/utils.py]: Nice! :tada: 

I thought the context manager will fail to remove temporary directories which are not empty, but it's not.

Comment (MartinBasti) [reply tests/integration/utils.py]: OR with pytest you can use https://docs.pytest.org/en/latest/tmpdir.html

But I'm fine with this

File: tox.ini (hunk 2 above)
Comment (MartinBasti) [reply tox.ini]: ok

File: tests/integration/utils.py (hunk 1 above)
Comment (csomh) [reply tests/integration/utils.py]: Even better :smiley: 

Deleting code is the best! :space_invader:

**Pull request merged**